back-end/controllers/signup.js

```diff
@@ -5,36 +5,31 @@ const User = require("../db/models/user")(db, Sequelize);
 
 
 const signup = (req, res) => {
-  const {email, password} = req.body;   //Sign up only needs email/pass
+  const {first_name, last_name, email, password, nick_names} = req.body;   //Sign up only needs email/pass

```

Comment:
Consider getting rid of extraneous //Sign up only... comment

Otherwise looks good